Knuth_Morris_Pratt_Algorithm/KMP.kt

```diff
@@ -0,0 +1,79 @@
+/*
+Knuth Morris Pratt Algorithm in Kotlin
+Author - Arkadip Bhattacharya (@darkmatter18)
```

Comment:
Please remove your author signature. It is an open source contribution and it can be seen in commits that you have contributed this.